src/Collapse/CollapseGroup/CollapseGroup.component.tsx

```diff
@@ -0,0 +1,83 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled from 'styled-components';
+// COMPONENTS
+import { Themes } from '../Collapse.component';
+
+// The mundane function of CollapseGroup is to remove the bottom border of any Collapse children,
+// save for the last one.
+const StyledCollapseGroup = styled.div`
+    .anchor-collapse:not(:last-child) {
```

Comment:
Optional: You can use a reference to styled components directly instead of like `.anchor-collapse`.
```css
${StyledCollapse}:not(:last-child) {
  
  & ${StyledCollapseButton} {
```

Reply:
I went this route to give an indication that there were a number of classes that a user could use to style the component.